core-test/src/test/scala/org/bitcoins/core/wallet/builder/BitcoinTxBuilderTest.scala

```diff
@@ -464,12 +466,23 @@ class BitcoinTxBuilderTest extends BitcoinSAsyncTest {
                                        fee,
                                        changeSPK._1,
                                        network)
-        val tx = Await.result(builder.flatMap(_.sign), 10.seconds)
-        assert(verifyScript(tx, creditingTxsInfo))
+        val txF = builder.flatMap(_.sign)
+
+        testRunFs += {
+          txF.map { tx =>
+            assert(verifyScript(tx, creditingTxsInfo))
+          }
+        }
+
+        succeed
     }
+
+    Future.sequence(testRunFs.result()).map(_.reduce((_, next) => next))
```

Comment:
So i think we should definitely move this stuff into `testkit` and actually write a test to make sure that this fails as expected? I think we can use this else where.